From 58429e75ca1d30791e8feae3155836850ad47615 Mon Sep 17 00:00:00 2001
From: insilications <boboniboni@gmail.com>
Date: Wed, 14 Oct 2020 19:29:11 -0300
Subject: [PATCH 3/4] Avoid stripping the -g option

---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 795b4c749..b028d4913 100755
--- a/configure.ac
+++ b/configure.ac
@@ -260,7 +260,6 @@ dnl **********************************************************************
 
 CURL_CHECK_COMPILER
 CURL_SET_COMPILER_BASIC_OPTS
-CURL_SET_COMPILER_DEBUG_OPTS
 CURL_SET_COMPILER_OPTIMIZE_OPTS
 CURL_SET_COMPILER_WARNING_OPTS
 
-- 
2.28.0

